CONTRIBUTING.md

```diff
@@ -119,24 +121,62 @@ To begin, install prerequisites:
 brew install wget
 ```
 
-`python` is the dependency manager for JVM based languages
+`maven` is the dependency manager for JVM based languages
```

Comment:
Can you remove all the references to JDK and Maven. As it is not needed for Remorph development any more.